lensless/recon.py

```diff
@@ -114,6 +114,26 @@ class :py:class:`~lensless.ReconstructionAlgorithm`. The three reconstruction
     python scripts/recon/admm.py -cn <CONFIG_FILENAME_WITHOUT_YAML_EXT>
 
 
+3D example
+----------
+
+It is also possible to reconstruct 3D scenes using Gradient Descent or APGD. ADMM doesn't supports 3D reconstruction yet.
+This requires to use a 3D PSF as an input in the form of a .npy file, which actually is a set of 2D PSFs corresponding to the same diffuser sampeled with light sources from different depths.
+The input data  for 3D reconstructions is still a 2D image, as collected by the camera. The reconstruction will be able to separate which part of the lensless data corresponds to which 2D PSF,
+and therefore to which depth, effectively generating a 3D reconstruction, which will be outputed in the form of a .npy file as well as a 2D projection on the depth axis to be displayed to the
+user as an image.
+
+As for the 2D ADMM reconstuction, scripts for 3D reconstruction can be found in ``scripts/recon/gradient_descent.py`` and ``scripts/recon/apgd_pycsou.py``.
```

Comment:
"As for the 2D ADMM....one depth level is used"

All this can be replaced with:

The same scripts for 2D reconstruction can be used for 3D reconstruction, namely ``scripts/recon/gradient_descent.py`` and ``scripts/recon/apgd_pycsou.py``. All that has to be done is set the corresponding input data and PSF via the command line:
```
EXAMPLE COMMAND
```